db/migrate/20181204000000_clear_invalid_invoices_from_orders.rb

```diff
@@ -0,0 +1,5 @@
+class ClearInvalidInvoicesFromOrders < ActiveRecord::Migration
+  def up
+    Order.where('invoice_id NOT IN (?)', Invoice.ids).update_all('invoice_id = NULL')
+  end
+end
```

Comment:
In migrations, one needs to define the models, because in later versions of the software the model may be renamed or otherwise not available anymore.
```ruby
class Order < ActiveRecord::Base; end
class Invoice < ActiveRecord::Base; end
```
may be enough. Or use SQL instead, which will always work with this database schema version.